backend/signing/configuration_test.go

```diff
@@ -0,0 +1,41 @@
+// Copyright 2020 Shift Crypto AG
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package signing
+
+import (
+	"testing"
+
+	"github.com/btcsuite/btcd/chaincfg"
+	"github.com/btcsuite/btcutil/hdkeychain"
+	"github.com/stretchr/testify/require"
+)
+
+func TestConfigurationsHash(t *testing.T) {
```

Comment:
Nice. Could you also add a Benchmark test? I'm slightly worried about time complexity. Just to make sure it doesn't take too long to compute the hash.

https://golang.org/pkg/testing/#hdr-Benchmarks

Reply:
uuuh seriously? Sounds like premature optimization. The hash function is gonna be called once per account, with at most three configs in the list. There is no way you could ever notice this, right?

Reply:
But there's no optimization. It's just a test.

Reply:
I was just curious to see ops/sec numbers, that's all.

Reply:
Don't you like benchmarks? They are awesome! :)

Reply:
Ok, I can live without the benchmark, I guess, but would've been happier with it.

Reply:
I mean I do like them, but not spending time on one if there is no need to make it faster :)